spec/models/requests/hold_item_spec.rb

```diff
@@ -111,12 +105,10 @@
         end
 
         it 'has the correct pick-up location id' do
-          stub_request(:get, stub_url)
-            .to_return(status: 200, body: responses[:get], headers: {})
-          stub_request(:put, stub_url)
-            .with(headers: { 'X-Accept' => 'application/xml' })
-            .to_return(status: 201, body: responses[:success], headers: {})
-          expect(hold_request.request_payload(submission.items.first)).to include("<pickup-location>333</pickup-location>")
+          stub_request(:post, stub_url)
+            .with(body: hash_including(request_type: "HOLD", pickup_location_type: "LIBRARY", pickup_location_library: "fcirc"))
```

Comment:
Did Alma actually accept the string "fcirc"? I was thinking we'd have to send that as "firestone".

Reply:
@kevinreiss No it did not, but that was the information in the mock.  I believe when we get real data (I hope) from the lama bibdata it will return firestone instead of fcirc